lib/MatchMediaController.ts

```diff
@@ -25,6 +25,7 @@ export class MatchMediaController implements ReactiveController {
   }
 
   hostConnected() {
+    this.evaluate();
```

Comment:
This fixes the problem of the mobile accordion not showing up on initial load.  The user would have to resize the screen to desktop and then back to mobile width to render the mobile accordion menu.

Reply:
WDYT about working with @zeroedin to replace this with screensize controller?